server/configs/test.conf

```diff
@@ -36,3 +36,7 @@ max_payload: 65536
 
 # slow consumer threshold
 max_pending_size: 10000000
+
+# ping interval and no pong threshold
```

Comment:
I meant "number of pings without a pong (pings with no pong)".
Please, let me know if you think there is a better yet brief description.